wm/screens.go

```diff
@@ -63,6 +63,24 @@ func (xsp *x11ScreensProvider) ScreenForWindow(win desktop.Window) *desktop.Scre
 	return xsp.screens[0]
 }
 
+func (xsp *x11ScreensProvider) ScreenForGeometry(x int, y int, width int, height int) *desktop.Screen {
+	if len(xsp.screens) > 1 {
```

Comment:
This logic is backwards I think, the quick exit is len(xsp.screens) == 1, so the rest of the function can be less indented.

Reply:
Fixed